packages/now-static-build/src/frameworks.ts

```diff
@@ -42,26 +40,33 @@ const frameworkList: Framework[] = [
     dependency: 'gatsby',
     buildCommand: 'gatsby build',
     getOutputDirName: async () => 'public',
-    defaultRoutes: async (dirPrefix: string) => {
-      try {
-        const nowRoutesPath = join(
-          dirPrefix,
-          'public',
-          '__now_routes_g4t5bY.json'
-        );
-        const content = await readFilePromise(nowRoutesPath, 'utf8');
-        const nowRoutes = JSON.parse(content);
-        try {
-          await unlinkPromise(nowRoutesPath);
-        } catch (err) {
-          // do nothing if deleting the file fails
-        }
-        return nowRoutes;
-      } catch (err) {
-        // if the file doesn't exist, we don't create routes
-        return [];
```

Comment:
I suggest returning the default routes you added here, and keep the old behavior if `__now_routes_g4t5bY.json` exists.